handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,57 @@
+# MinIO as storage provider for the FileService
+
+SystemLink FileService allows you to configure and use the Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the FileService also works with MinIO, a server-side software storage stack that is compatible with Amazon S3.
+
+To use MinIO with SystemLink, you need SystemLink version 2020 R3 or later.
+
+To use MinIO as a storage provider, set up the MinIO server and configure the File Service to use that server.
+
+## Running MinIO server
+
+1. In order to run the MinIO server, download the server from the [MinIO website](https://min.io/download). From the command line, run the following command to run the server:
```

Comment:
For minimalism, can take out 'In order to run the MinIO server,'

Reply:
That raises the question if I can also modify the two sentences just above of that line which both start in "To use MinIO". I feel like my English is limited to just one way of starting a sentence :D 

Any suggestions? I think at least one of the sentence should read differently.

Reply:
I left it in the other instances, since it's not a step but more of an overview of what the user generally needs to do. So I think you're fine!